tendrl-frontend.spec

```diff
@@ -1,38 +1,33 @@
-%define pkg_name tendrl-frontend
-%define pkg_version 0.0.1
-%define pkg_release 1
-
-Name: %{pkg_name}
-Version: %{pkg_version}
-Release: %{pkg_release}%{?dist}
+Name: tendrl-frontend
+Version: 1.1
+Release: 1%{?dist}
 BuildArch: noarch
 Summary: GUI for Tendrl
 License: ASL 2.0
 Group:   Applications/System
-Source0: %{pkg_name}-%{pkg_version}.tar.gz
-BuildRoot: %{_tmppath}/%{pkg_name}-%{pkg_version}-%{pkg_release}-buildroot
-Url: https://github.com/Tendrl/tendrl_frontend
+Source0: %{name}-%{version}.tar.gz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{pkg_release}-buildroot
+URL: https://github.com/Tendrl/tendrl_frontend
 
-Requires: tendrl
+Requires: tendrl-api
 
 %description
 Contains the JavaScript GUI content for the Tendrl front-end components
 (dashboard, login screens, administration screens)
 
 %prep
-%setup -n %{pkg_name}-%{pkg_version}
+%setup
 
 %install
-rm -rf $RPM_BUILD_ROOT
-install -m 755 -d $RPM_BUILD_ROOT/%{_datadir}/Tendrl/webapp
-cp -r ./dist/* $RPM_BUILD_ROOT/%{_datadir}/Tendrl/webapp/
+install -m 755 -d $RPM_BUILD_ROOT/%{_datadir}/tendrl/webapp
+cp -r ./dist/* $RPM_BUILD_ROOT/%{_datadir}/tendrl/webapp/
```

Comment:
As we discussed, does this work after RPM installation? After installation, user not supposed to do any manual copying of files to /var/www/html